README.adoc

```diff
@@ -39,16 +39,18 @@ contains new data. If so the application will the reindex the Elasticsearch data
 == Run Local Plugin Site API
 
 ----
+GITHUB_CLIENT_ID="dummy"
```

Comment:
From an infrastructure standpoint, is this expected to be a personal access token in the future or should we be thinking about making a GitHub App with a client id, secret, etc for this?

Reply:
It's supposed to be a client id of a GitHub App, but the app does not need to have any callback URL, just the client ID and secret.